tests/src/testModels.js

```diff
@@ -157,6 +157,7 @@ describe('Model', function() {
         expect(permissions.features_devel).toBe(true);
         expect(permissions.features_multiarch).toBe(true);
         expect(permissions.features_canbus).toBe(true);
+        expect(permissions.features_per_app_dev_shm).toBe(true);
```

Comment:
For this to actually work, you need to add this permission to the test permission files. Check which appId is used by this test to find the proper file.